vertical_tail.py

```diff
@@ -398,25 +402,34 @@ def setup(self, vt, state, fitDrag):
         Dvt    = Variable('D_{vt}', 'N', 'Vertical tail viscous drag, cruise')
         Rec    = Variable('Re_{vt}', '-', 'Vertical tail reynolds number, cruise')
         CDvis  = Variable('C_{D_{vis}}', '-', 'Viscous drag coefficient')
-
+        dxlead  = Variable('\\Delta x_{lead_{vt}}', 'm',
+                           'Distance from CG to vertical tail leading edge')
+        dxtrail = Variable('\\Delta x_{trail_{vt}}', 'm',
+                           'Distance from CG to vertical tail trailing edge')
+ 
         #constraints
         constraints = []
 
          #vectorized constraints
-        constraints.extend([
-                    
-##            TCS([CLvt*(1 + clvt/(np.pi*self.vt['e_v']*self.vt['A_{vt}'])) <= clvt]),
+        with SignomialsEnabled():
+            constraints.extend([
+                        
+    ##            TCS([CLvt*(1 + clvt/(np.pi*self.vt['e_v']*self.vt['A_{vt}'])) <= clvt]),
 
-            # Finite wing theory
-            # people.clarkson.edu/~pmarzocc/AE429/AE-429-4.pdf
-            # Valid because tail is untwisted and uncambered
-            # (lift curve slope passes through origin)
+                # Finite wing theory
+                # people.clarkson.edu/~pmarzocc/AE429/AE-429-4.pdf
+                # Valid because tail is untwisted and uncambered
+                # (lift curve slope passes through origin)
 
-            Dvt >= 0.5*state['\\rho']*state['V']**2*self.vt['S_{vt}']*CDvis,
+                Dvt >= 0.5*state['\\rho']*state['V']**2*self.vt['S_{vt}']*CDvis,
 
-            # Cruise Reynolds number
-            Rec == state.atm['\\rho']*state['V']*self.vt['\\bar{c}_{vt}']/state.atm['\\mu'],
-            ])
+                # Cruise Reynolds number
+                Rec == state.atm['\\rho']*state['V']*self.vt['\\bar{c}_{vt}']/state.atm['\\mu'],
+
+               # Moment arm and geometry -- same as for htail
+              dxlead + self.vt['c_{root_{vt}}'] <= dxtrail,
+              TCS([dxlead + self.vt['y_{\\bar{c}_{vt}}']*self.vt['\\tan(\\Lambda_{vt})'] + 0.25*self.vt['\\bar{c}_{vt}'] >= self.vt['l_{vt}']],reltol=1e-2), # [SP]
+              ])
 
         if fitDrag:
             constraints.extend([
```

Comment:
@1ozturkbe something went wrong here, 777 model is broken by this. Seems to work for 737 though. Need to fix this ASAP.